playbooks/deploy_edpm_os_install.yml

```diff
@@ -56,9 +56,21 @@
         tasks_from: install.yml
       tags:
         - edpm_chrony
+    - name: install frr
```

Comment:
Task name should be start with an uppercase letter (see https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/name.py#L127)